src/textual/_types.py

```diff
@@ -1,13 +1,12 @@
-from typing import Awaitable, Callable, List, TYPE_CHECKING, Union
+from typing import Callable, List, Union
+from typing import Awaitable, TYPE_CHECKING
 
-from rich.segment import Segment
```

Comment:
Is this removal on purpose? There's reference to `Segment` below and I get a type error for this.

Reply:
Fixed